test/scripts/start-services.sh

```diff
@@ -9,7 +9,7 @@ start_ganache()
     echo "Starting Ganache"
     yarn run ganache-cli \
         --port=8545 \
-        --blockTime=0 \
+        --blockTime=15 \
```

Comment:
Is this strictly necessary? Its very convenient for the E2E tests to have blocks mined instantly after each transaction. Having a 15second delay means we'll need to update our tests for this change in behavior. And the tests will run slower too as they will need to wait up to 30s after submitting each transaction

Reply:
The relayer relies on a stream of incoming blocks to figure out what the latest block is. Otherwise it gets stuck in a loop retrieving a next header that doesn't exist yet. I'm going to overhaul this implementation, but in the mean time, would it be fast enough to set block time to 1 second?

Reply:
ah I see. Makes sense. I think its OK then and I'll update the tests to wait 20-30s after each transaction.